evap/results/tests/test_views.py

```diff
@@ -27,6 +27,36 @@ def setUpTestData(cls):
         cls.semester = mommy.make(Semester, id=1)
 
 
+class TestResultsViewContributionWarning(WebTest):
+
+    @classmethod
+    def setUpTestData(cls):
+        cls.semester = mommy.make(Semester, id=3)
+        mommy.make(UserProfile, username='contributor', groups=[Group.objects.get(name='Staff')], email="contributor@institution.example.com")
```

Comment:
is that user supposed to be a contributor or staff user?

Reply:
I split their personality.